cfme/tests/infrastructure/test_infra_quota.py

```diff
@@ -110,10 +112,8 @@ def template_name(provisioning):
 @pytest.fixture(scope="function")
 def provisioner(request, setup_provider, provider):
     def _provisioner(template, provisioning_data, delayed=None):
-        sel.force_navigate('infrastructure_provision_vms', context={
-            'provider': provider,
-            'template_name': template,
-        })
+        vm = Vm(name=vm_name, provider=provider, template_name=template)
```

Comment:
actually, this should be implemented as a vm.deploy(blabla) method but it will take time to get rid of all the fixtures and functions.